Nef_S2/include/CGAL/Nef_S2/Sphere_circle.h

```diff
@@ -78,16 +81,22 @@ create any great circle that contains $p$ and $q$.}*/
   }
 }
 
- Sphere_circle(const Plane_3& h) : Base(h)
+Sphere_circle(const Plane_3& h) : Base(h)
 /*{\Mcreate creates the circle of $S_2$ corresponding to the plane
 |h|. If |h| does not contain the origin, then |\Mvar| becomes the
 circle parallel to |h| containing the origin.}*/
 {
-  if(h.d() != 0) *this = Plane_3(h.a(),h.b(),h.c(),RT(0));
+  if(!is_zero(h.d()))
+    *this = Plane_3(h.a(),h.b(),h.c(),RT(0));
 }
 
-Sphere_circle(const RT& x, const RT& y, const RT& z): Base(x,y,z,0) {}
+Sphere_circle(const Origin& o, const Vector_3& v) : Base(o,v) {}
```

Comment:
@mglisse I should probably do the same in my own PR then. i.e `Origin o` instead of `const Origin&` o?

Reply:
Indeed. If the function is inlined, both are equivalent. And if it isn't, the cost is negligible. Passing tag types by value is a nicer default when you have the choice. But note that we often end up passing tag types by reference, because they go through generic forwarders, and that's ok, not worth complicating the code.

Reply:
I guess I just wanted the code to be consistent, but I see `const Origin&` elsewhere, so I guess it doesn't matter.